designer/submodules/packages/components/src/components/usages/nodesCell.tsx

```diff
@@ -19,35 +20,40 @@ export const NodesCell = ({ filterText, ...props }: GridRenderCellParams & { fil
         [filterSegments],
     );
 
-    const sorted = useMemo(() => value.map((node) => [countMatches(node), node]).sort(([a], [b]) => b - a), [countMatches, value]);
+    const sorted = useMemo(() => value.map((node: NodeUsageData) => [countMatches(node.nodeId), node.nodeId, node.fragmentNodeId, node.type]).sort(([a], [b]) => b - a), [countMatches, value]);
 
-    const elements = sorted.map(([match, node]) => (
-        <NodeChip key={node} icon={icon} node={node} filterText={filterText} rowId={id} matched={filterText ? match : -1} />
+    const elements = sorted.map(([match, nodeId, fragmentNodeId, nodeType]) => (
```

Comment:
This order is correct? We want to sort by nodeId / fragmentNodeId instead of fragmentNodeId / nodeId? We you have needed to add nodeType as a one more orderBy element?

Reply:
I corrected that. Also I removed `nodeType` from it and replaced it with simple `null`-check. I don't think such a data pass into component is good design choice, because every new data is currently taken into consideration by sorting function.